### DISCUSSION TITLE
Doublets

### DISCUSSION THREAD

**Discussion by Guliariel** [GreenleafLab/ArchR#1835] [Questions / Documentation]
Hello,
I am looking to get the list of the excluded cells/barcodes after Doublets filtering.
Where can I get this information from?
Thank you!
ArchR is such a great tool :)

**Comment by rcorces** [comment]
if you have already run `filterDoublets()` then the cells that were found to be doublets will have been removed from `cellColData` so you cant see them any more.
Prior to running `filterDoublets()` you just need to check `cellColData` to see which cells dont meet the criteria (`cutEnrich`, `cutScore`, `filterRatio`)  you will using during `filterDoublets()`. You can always read the code in `filterDoublets` to understand this better.
https://github.com/GreenleafLab/ArchR/blob/968e4421ce7187a8ac7ea1cf6077412126876d5f/R/FilterCells.R#L45-L106
